src/screens/flows/Send/components/ReviewSend/index.js

```diff
@@ -120,14 +117,12 @@ const ReviewSend = ({
         {token && (
           <Row style={styles.row}>
             <Column>
-              {value ? (
-                <Text style={styles.title}>${value?.display}</Text>
-              ) : null}
-              <Text
-                type="caption"
-                style={value ? styles.subtitle : styles.title}>
+              <Text type="caption" style={styles.title}>
                 {`${amount?.display} ${token.symbol}`}
               </Text>
+              {value ? (
+                <Text style={styles.subtitle}>${value?.display}</Text>
+              ) : null}
```

Comment:
Does this work?, or is it a reason why the ternary with null?
```suggestion
              {value && (
                <Text style={styles.subtitle}>${value?.display}</Text>
              )}
```

Reply:
Sometimes using && causes crashes in renders, mostly when using Text component. So i ended using always the ternary. The last crash i solved was in the transaction item, one specific transaction made the app crash a few months ago.

Reply:
Great! Didn't know that